src/web/components/SignInGate/SignInGateSelector.tsx

```diff
@@ -151,23 +117,7 @@ export const SignInGateSelector = ({
 	CAPI,
 }: SignInGateSelectorProps) => {
 	const [showGate, setShowGate] = useState(true);
-	const [currentTest, setCurrentTest] = useState<CurrentABTest>({
-		name: '',
-		variant: '',
-		id: '',
-	});
-
-	const ab = useAB();
-
-	useEffect(() => {
```

Comment:
👍  I see we don't need this useEffect hook and you've moved this into `useSignInGateSelector`